ensembl/htdocs/info/genome/compara/family.html

```diff
@@ -4,6 +4,12 @@
 </head>
 
 <body>
+<dialog open>
+<h2>Compara gene and protein families were retired as of release 102, as announced in a <a href="https://www.ensembl.info/2020/11/24/gene-and-protein-families-retiring/">November 2020 Ensembl blog post</a>.</h2>
+  <form method="dialog">
+  <center><button>Close</button></center>
+  </form>
+</dialog>
```

Comment:
This presents the message loud and clear. But I was wondering, could it be made more consistent with other Ensembl web code in some way, such as by using a `div`-based `info` box like [the one describing the taxonomic group(s) for which Orthology QC is done](https://github.com/thiagogenez/public-plugins/blob/7a5cb7967fa03a4602f92c701bb33add4dacaa88/ensembl/htdocs/info/genome/compara/Ortholog_qc_manual.html#L47-L54) (which can be seen on the [Orthology QC webpage](https://jul2022.archive.ensembl.org/info/genome/compara/Ortholog_qc_manual.html))?

Reply:
i agree with Thomas - this is not Ensembl-like HTML, and goes against our style gudelines for the site. We also don't really want to be able to dismiss the message - if this page getting cached, another user might not see it.

Please change this to use the standard "info" class on a div element.

Reply:
Thanks, @twalsh-ebi and @ens-ap5 for highlighting this issue. Fixed!